cpp/src/metrics/trustworthiness.cu

```diff
@@ -105,79 +37,18 @@ namespace ML {
         template<typename math_t, DistanceType distance_type>
         double trustworthiness_score(const cumlHandle& h, math_t* X,
                             math_t* X_embedded, int n, int m, int d,
-                            int n_neighbors)
-        {
-            const int TMP_SIZE = MAX_BATCH_SIZE * n;
-
+                            int n_neighbors) {
```

Comment:
not sure about docstring and template parameter so wanted to ask: `distance_type` is not a function parameter but a template one, is there a way to indicate that in the docstring?